README.md

```diff
@@ -23,11 +31,11 @@ if(process.env.NODE_ENV === 'development') {
 }
 ```
 
-and you'll be away laughing.
+
 
 ## Configuration
 
-The overlay works by using a WebSocket that connects to `webpack-serve` à la `webpack-hot-client`.  
```

Comment:
The double space here was deliberate, as it's an indicator in markdown that the following line should be on a newline. Removing it causes the next line to be rendered on the same line as this line.

(I'm guessing your formatter isn't a markdown-aware one?)

Reply:
Oh right, I forget that's a really unintuitive feature of Markdown (near invisible trailing whitespace used for actual formatting); I tend to use `<br>` tags instead. 

(I guess not :/ will have to see if there's a better plugin for VSCode)